.github/workflows/python-app.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python application
```

Comment:
Can we simplify away the not needed parts of this example? Also fill in better names?

E.g. rename `python-app.yaml` to `python-tests.yaml` 

build -> CI or run tests

Reply:
will do this later, when I'll do a python package, see #4